modules/oadp-monitoring.adoc

```diff
@@ -0,0 +1,378 @@
+
+.Procedure
+
+. Create a PrometheusRule YAL file with the sample `OADPBackupFailing` alert and save it under `4_create_oadp_alert_rule.yaml` name.
+. In this example an alert displays when there is an increase of failing backups, which indicates new failures, during the 2 last hours that is greater than 0 and this state persisted for at least 5 minutes. If the time of the first increase is less than 5 minutes, the Alert will be in a `Pending` state, after which it will turn into a `Firing` state.
++
+[source,yaml]
++
+----
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: sample-oadp-alert
+  namespace: openshift-adp
+spec:
+  groups:
+  - name: sample-oadp-backup-alert
+    rules:
+    - alert: OADPBackupFailing
+      annotations:
+        description: 'OADP had {{$value | humanize}} backup failures over the last 2 hours.'
+        summary: OADP has issues creating backups
+      expr: |
+        increase(velero_backup_failure_total{job="openshift-adp-velero-metrics-svc"}[2h]) > 0
+      for: 5m
+      labels:
+        severity: warning
+----
++
+. Apply the `4_create_oadp_alert_rule.yaml`, this creates the PrometheusRule in the `openshift-adp namespace`.
++
+[source,terminal]
+----
+$ oc apply -f 4_create_oadp_alert_rule.yaml
+prometheusrule.monitoring.coreos.com/sample-oadp-alert created
+----
++
+. After the Alert is triggered, it can be viewed in the following ways:
+.. In the *Administrator* view under *Observe/Alerting* menu. You must select `User` in the `Filter` box, otherwise by default only the `Platform` alerts are displayed.
+.. In the *Developer* view, select the *Observe* menu.
++
+.OADP backup failing alert
+
+image::oadp-backup-failing-alert.png[OADP backup failing alert]
+
+[id="List-of-metrics_{context}"]
+==  List of available metrics
+
+These are the list of metrics provided by the OADP together with their https://prometheus.io/docs/concepts/metric_types/[Types].
+
+|===
+|Metric Name |Description |Type
+
+|kopia_content_cache_hit_bytes
+|Number of bytes retrieved from the cache
+|Counter
+
+|kopia_content_cache_hit_count
+|Number of time content was retrieved from the cache
+|Counter
+
+|kopia_content_cache_malformed
+|Number of times malformed content was read from the cache
+|Counter
+
+|kopia_content_cache_miss_count
+|Number of time content was not found in the cache and fetched
+|Counter
+
+|kopia_content_cache_missed_bytes
+|Number of bytes retrieved from the underlying storage
+|Counter
+
+|kopia_content_cache_miss_error_count
+|Number of time content could not be found in the underlying storage
+|Counter
+
+|kopia_content_cache_store_error_count
+|Number of time content could not be saved in the cache
+|Counter
+
+|kopia_content_get_bytes
+|Number of bytes retrieved using GetContent
+|Counter
+
+|kopia_content_get_count
+|Number of time GetContent() was called
```

Comment:
```suggestion
|Number of times `GetContent()` was called
```

or

```suggestion
|Number of times the `GetContent()` function was called
```

Don't know if "function" would be the right descriptor here.

Same suggestions for the other `GetContent()` and `WriteContent()` entries below.